README.md

```diff
@@ -563,8 +563,14 @@ end
 Then you can get this error message in exception handler:
 ```ruby
 rescue_from Pundit::NotAuthorizedError do |e|
-  message = e.reason ? I18n.t("pundit.errors.#{e.reason}") : e.message
-  flash[:error] = message, scope: "pundit", default: :default
+  message = 
+    if e.reason 
+      I18n.t("pundit.errors.#{e.reason}", scope: "pundit", default: :default)
```

Comment:
Is `scope` required? Isn't it covered by starting the key with `pundit`?

Is the `default:` an essential part of this approach? It seems a bit odd to specify a default and then to also provide a default in the `else` clause? I wonder if we can just remove this and go back to the ternary `if`?

Reply:
Correct, I missed it. Just tested right now

Reply:
I agree about this if, fixed it and tested